__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,20 @@
+const Port = require('../src/Port.js');
+const Ship = require('../src/Ship.js');
+const Itinerary = require('../src/Itinerary.js');
```

Comment:
The thing you're missing here is isolating your tests as we did in our labs.
So, you should aim to write your tests in a way that you make use of dummies, spys, etc. so you're not actually bring functionality of another component to this set of tests.